src/view/com/util/Link.tsx

```diff
@@ -166,8 +169,14 @@ export const TextLink = observer(function TextLink({
     return {}
   }, [href])
 
+  React.useEffect(() => {
+    // @ts-expect-error web only, shows date on hover -esb
+    anchorRef.current?.setNativeProps?.({title})
```

Comment:
Oof, is this really the only way to do this?

Reply:
Yeah the `title` attr is ignored afaict :((((((

Reply:
I almost wonder if we'd be better off putting it in a `data-` attribute and adding a css-only tooltip library

Reply:
Also just remembered I'd heard that the new arch will deprecate `setNativeProps`. This is still a first party API, so could work for now, but yeah I can do a simple CSS tooltip too nbd.